src/commands/create.js

```diff
@@ -85,8 +87,25 @@ export default asyncCommand({
 		}
 		catch (err) {}
 
+		if (exists && argv.force) {
+			const question = {
+        type: 'confirm',
+        name: 'enableForce',
+        message: `You are using '--force'. Do you wish to continue?`,
+        default: true,
+      };
+
+			let forceInit = await inquirer.prompt(question);
+
+			if (forceInit.enableForce) {
+				process.stdout.write('Initializing project in the current directory...\n');
+			} else {
+				throw Error('Cannot initialize the project in the current directory');
+			}
+		}
+
 		if (exists && !argv.force) {
-			throw Error('Directory already exists.');
+			throw Error('Cannot intialize in the current directory, please specify a different destination.');
```

Comment:
Oh sure, not a problem at all. Red color?

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_